docs/docs/creating-a-transformer-plugin.md

```diff
@@ -158,20 +160,46 @@ function transformObject(obj, id, type) {
     ...obj,
     id,
     children: [],
-    parent: node.id,
+    parent: null,
     internal: {
       contentDigest: createContentDigest(obj),
       type,
     },
   }
   createNode(yamlNode)
-  createParentChildLink({ parent: node, child: yamlNode })
 }
 ```
 
 Above, you create a `yamlNode` object with the shape expected by the [`createNode` action](/docs/actions/#createNode).
 
-You then create a link between the parent node (file) and the child node (yaml content).
+#### Creating the transformer relationship
```

Comment:
A lot of this content moved from the source plugin guide because it felt more relevant here, I took the liberty of adding some more headers and breaking this step above into the places it looked like it belonged.